dev/tests/functional/standalone_bootstrap.php

```diff
@@ -53,6 +53,9 @@
     defined('WAIT_TIMEOUT') || define('WAIT_TIMEOUT', 30);
     $env->setEnvironmentVariable('WAIT_TIMEOUT', WAIT_TIMEOUT);
 
+    defined('VERBOSE_ARTIFACTS') || define('VERBOSE_ARTIFACTS', false);
```

Comment:
The logic of VERBOSE_ARTIFACTS flag is not quite right. Maybe you should use `string` instead of `boolean`.

Reply:
Hm, where does this get weird, in the comparisons in the file below? What behavior are you seeing?

Reply:
I found the flag is not working as expected and it turns out VERBOSE_ARTIFACTS is empty string.

Reply:
I see that, only happens if the flag is not present at all. I fixed this by nakign it set `false` instead, and changed the string comparison in the below file.